Library/Formula/blackbox.rb

```diff
@@ -0,0 +1,36 @@
+class Blackbox < Formula
+  desc "Safely store secrets in Git/Mercurial/Subversion"
+  homepage "https://github.com/StackExchange/blackbox"
+  url "https://github.com/StackExchange/blackbox/archive/v1.20160122.tar.gz"
+  sha256 "ac5de1d74fdbe88604b34949f3949e53cb72e55e148e46b8c2be98806c888a10"
+
+  depends_on "gpg"
+
+  def install
+    bin.install "bin/_blackbox_common.sh"
+    bin.install "bin/_blackbox_common_test.sh"
+    bin.install "bin/_stack_lib.sh"
```

Comment:
Hi @mikemcquaid, there is a Makefile in the bin/ directory in the tarball so I think selecting the actual files would be better. 

Most of the scripts require the files such as _blackbox_common.sh to be in the same directory as the script. Does homebrew take care of this with symlinking?